_pages/algorithmia-enterprise/scms.md

```diff
@@ -0,0 +1,113 @@
+
+__SSH URL__: This is the URL by which we will attempt to pull source code from your GitHub Instance. For public GitHub, this value should be ssh://git@github.com. For GitHub Enterprise instances, replace the domain name with the domain for your instance. For example, if your GitHub Enterprise instance lives at https://example.com, you should enter ssh://git@example.com. 
+
+__OAuth Client ID__: This is the value of the “Client ID” you received during the “Create a Github OAuth App” step. 
+
+__OAuth Client Secret__: This is the value of the “Client Secret” you received during the “Create a Github OAuth App” step. 
+
+Once you’re satisfied with your GitHub SCM’s configuration, click “Connect SCM”. After a brief moment, you should receive a message informing you that your SCM was created successfully.
+
+#### Creating Algorithms with Github
+
+You can read more about creating algorithms with Github in our [user documentation](/developers/algorithm-development/source-code-management).
+
+### Managing SCMs
+
+Both Algorithmia and Github SCMs can be enabled and disabled to your liking, ensuring that your users leverage the correct SCM. Additionally, you can select a “default” SCM for your Algorithmia instance, in the case that you wish to have multiple SCMs enabled at once.
+
+#### Disabling an SCM
+
+By default, all new SCMs are enabled for use in your cluster. You can easily change this by navigating to the admin source control management page, clicking on the SCM you wish to disable, and clicking “Disable”:
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/enable_scm.png" alt="Set an SCM as default" class="screenshot img-sm">
+
+This will not result in breakage or removal of any algorithms created with that SCM. Disabling an SCM solely prevents new algorithms from being created that leverage the particular SCM.
+
+#### Enabling an SCM
+
+If you wish to enable an SCM, simply navigate to the admin source control management page, select the SCM you wish to enable, and click “Enable”.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/disable_scm.png" alt="Set an SCM as default" class="screenshot img-sm">
+
+The SCM will now be available for use by all users within your Algorithmia instance.
+
+#### Selecting a Default SCM
+
+If you have multiple enabled SCMs, you may wish to guide your users to a specific SCM, such as Github. You can do this by selecting an enabled SCM and clicking the “Make Default” button. Once made default, your selected SCM will be the first SCM selected whenever a user attempts to create an algorithm.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/make_scm_default.png" alt="Set an SCM as default" class="screenshot img-sm">
```

Comment:
All of the images on this page use the same `alt` tag value of `Set an SCM as default`

Reply:
Good catch! Updating

Reply:
@bnlb Fixed!